homeassistant/components/edl21/translations/de.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please don't add other languages, only English (`strings.json` and `en.json`) others can later be translated via the Lokalise translation platform.